pkg/cmd/image/list.go

```diff
@@ -405,3 +412,21 @@ func unpackedImageSize(ctx context.Context, s snapshots.Snapshotter, img contain
 	}
 	return usage.Size, nil
 }
+
+// https://github.com/moby/moby/blob/65c371ed6e48d0014a4826a6933f9231f688327a/daemon/containerd/image_list.go#L184-L194
+// What is VirtualSize: https://github.com/docker/docs/issues/1520#issuecomment-305179362
+func computeVirtualSize(ctx context.Context, s snapshots.Snapshotter, img containerd.Image) (int64, error) {
+	chainIDs, err := img.RootFS(ctx)
```

Comment:
diffIDs, not chainID 
https://github.com/opencontainers/image-spec/blob/v1.1.0-rc2/config.md#layer-chainid